client/layout/main-banner/index.js

```diff
@@ -0,0 +1,31 @@
+import { Gridicon } from '@automattic/components';
+import './style.scss';
+
+export function MainBanner( { sectionName, siteIntent, siteSlug } ) {
+	const allowList = [ 'posts', 'media', 'pages', 'home' ];
+	if ( ! allowList.includes( sectionName ) ) {
+		return null;
+	}
+	return (
+		<div className="layout__launchpad-banner">
+			<div className="layout__launchpad-banner-content">
+				🚀
+				<p>
+					Finish your onboarding in the Launchpad! Return by clicking
+					<a href={ `/setup/${ siteIntent }/launchpad?siteSlug=${ siteSlug }` }> here</a>
+				</p>
+			</div>
+			<button
```

Comment:
Did we intend to use `Button` here? Seems like a good candidate for that. Especially, the one from `@wordpress/components` supports both a label and an icon.